api/tests/unit/domain/usecases/authenticate-pole-emploi-user_test.js

```diff
@@ -69,7 +82,7 @@ describe('Unit | Application | Use Case | authenticate-pole-emploi-user', () =>
 
   context('When user does no exist yet', () => {
```

Comment:
context('When user does no exists yet', () => {

Reply:
Je ne suis pas sûr de ça. Par contre j'ai remplacé `no` par `not`.